CHANGELOG

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------------------
+1.1.0
+* Introduce support for event tags.
+* Add optional eventTags argument to track method signature.
+* Removed optional eventValue argument from track method signature.
```

Comment:
I'd say we are deprecating it. You can technically still pass it in but you'll get a warning